pkg/mosn/mosn.go

```diff
@@ -127,7 +128,7 @@ func (m *Mosn) inheritConfig(c *v2.MOSNConfig) (err error) {
 	server.EnableInheritOldMosnconfig(c.InheritOldMosnconfig)
 
 	// default is graceful mode, turn graceful off by set it to false
-	if !c.DisableUpgrade && server.IsReconfigure() {
+	if !m.ReloadProcess && !c.DisableUpgrade && server.IsReconfigure() {
```

Comment:
maybe we can just reuse the `DisableUpgrade`? just set the `DisableUpgrade` flag instead.

Reply:
DisableUpgrade is got from the config.json file. Do you mean we just change disable_upgrade directly?

Reply:
yes, overwrite the `DisableUpgrade` after getting `c` from `config.json`.

Reply:
> yes, overwrite the `DisableUpgrade` after getting `c` from `config.json`.

okay.